src/rest-server/src/models/job.js

```diff
@@ -85,6 +85,8 @@ class Job {
       case 'TASK_COMPLETED':
         if (exitCode === 0) {
           return 'SUCCEEDED';
+        } else if (exitCode === -7400) {
```

Comment:
Show on web?

Reply:
web portal changes will be in another pull request